mon-pix/app/controllers/fill-in-certificate-verification-code.js

```diff
@@ -0,0 +1,38 @@
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Controller from '@ember/controller';
+
+export default class FillInCertificateVerificationCode extends Controller {
+  @service store;
+
+  certificateVerificationCode = null;
+
+  codeRegex = '^[P,p]-[2346789BCDFGHJKMPQRTVWXY]{6}-[0-9]{3}$';
```

Comment:
On peut creer ici la Regex histoire de pas le refaire à chaque check
```suggestion
  codeRegex = new RegExp('^[P,p]-[2346789BCDFGHJKMPQRTVWXY]{6}-[0-9]{3}$', 'gi');
```